docs/tutorial/topics/custom-icons.rst

```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
 Let's provide some actual icons. Download :download:`this icons.zip bundle
```

Comment:
Are we up for a `code-block:: console` here to show how to use `curl` or `wget` to download the bundle (Linux, macOS) and `unzip` to extract it? Would we do that as part of this PR, or a subsequent PR?

Reply:
I've experimented with trying to add extra tables for new code-blocks yesterday and ran into many issues. I think this should be resolved in a subsequent PR.